frontends/web/src/routes/account/send/send.tsx

```diff
@@ -225,14 +225,8 @@ class Send extends Component<Props, State> {
   };
 
   private handleKeyDown = (e: KeyboardEvent) => {
-    if (e.keyCode === 27) {
-      if (this.state.activeScanQR) {
-        this.toggleScanQR();
-        return;
-      }
-      if (!this.state.activeCoinControl) {
-        route(`/account/${this.props.code}`);
-      }
+    if (e.keyCode === 27 && !this.state.activeCoinControl && !this.state.activeScanQR) {
+      route(`/account/${this.props.code}`);
```

Comment:
This has been existing "forever" (not regression). Did it in this PR anyway since it's small and still related to Dialog. :)